sbom-generator/scripts/generate-sbom.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+function log() {
+    msg="$1"
+    shift
+    # shellcheck disable=SC2059 # "$1" for printf is assumed safe.
+    printf "$msg\n" "$@" >&2
+}
+
+# Add extra logging if the runner was run with debug logging.
+test -z "${RUNNER_DEBUG+x}" || set -x
```

Comment:
Can you shortly explain how can we set this variable in pipelines? I think we would need write action.yaml accordingly, otherwise this never going to be set (or alternatively I am just missing something obvious here)

Reply:
That's an environment variable automatically set by GitHub:
- `RUNNER_DEBUG` is blank when not running GitHub workflows in debug mode
- It's set to `1` (i.e., non-blank) otherwise

Debug mode enabled like so by rerunning workflows:

<img width="500" alt="Screenshot 2024-09-27 at 2 19 59 PM" src="https://github.com/user-attachments/assets/1709220b-eab6-4224-8985-c42e003effed">

Reply:
TIL, thanks!!